client/app/bundles/course/duplication/reducers/duplication.js

```diff
@@ -112,10 +113,10 @@ export default function (state = initialState, action) {
     }
 
     case actionTypes.DUPLICATE_COURSE_REQUEST:
-    case actionTypes.DUPLICATE_ITEMS_REQUEST: {
+    case actionTypes.DUPLICATE_ITEMS_REQUEST:
+    case actionTypes.DUPLICATE_COURSE_SUCCESS: {
```

Comment:
Hmm this change doesn't make sense to me. If it succeeded, the isDuplicating should be `false`?

Reply:
Actually there is a small time gap between duplication success and redirection to the new course. So to keep the circular progress going, I set the isDuplicating to true. Let me know what you think.

Reply:
I think if that's the case, there should be a third state, perhaps tracked by another `isSuccess` boolean.

Then, upon succeeding, the spinner changes to a message like "Successfully duplicated. Redirecting you to the new course now...", with the button still disabled.

Reply:
haha right. actually I have those - let me change it!